modules/BiotypeQC.nf

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env nextflow
+
+/*
+BioQC is a is a R/Bioconductor package to detect tissue heterogeneity in gene expression data
+*/
+
+params.gene_expression = "/home/ausserh/myScratch/Salmon_test.txt"
+params.samplesheet = "/home/ausserh/testMatrix.tsv"
+params.marker = "extdata/exp.tissuemark.affy.roche.symbols.gmt"
+
+
+process 'BioQC' {
+
+  conda 'bioconductor-bioqc r-readr'
```

Comment:
This should read as:
conda 'conda-forge::r-base=4.0 conda-forge::r-readr bioconda::bioconductor-bioqc'